src/wfpt.pyx

```diff
@@ -61,14 +61,267 @@ def wiener_like(np.ndarray[double, ndim=1] x, double v, double sv, double a, dou
     for i in range(size):
         p = full_pdf(x[i], v, sv, a, z, sz, t, st, err, n_st, n_sz, use_adaptive, simps_err)
         # If one probability = 0, the log sum will be -Inf
+        #print('p: ',p)
         p = p * (1 - p_outlier) + wp_outlier
+        #print('after p: ',p)
         if p == 0:
             return -np.inf
 
         sum_logp += log(p)
 
     return sum_logp
 
+def wiener_like_rlddm(np.ndarray[double, ndim=1] x, 
+                      np.ndarray[long, ndim=1] response,
+                      np.ndarray[double, ndim=1] feedback,
+                      np.ndarray[long, ndim=1] split_by,
+                      double q,
+                      double alpha, double dual_alpha, double v, double sv, double a, double z, double sz, double t,
+                      double st, long uncertainty, double err, int n_st=10, int n_sz=10, bint use_adaptive=1, double simps_err=1e-8,
+                      double p_outlier=0, double w_outlier=0):
+    cdef Py_ssize_t size = x.shape[0]
+    cdef Py_ssize_t i
+    cdef Py_ssize_t s_size
+    cdef int s
+    #cdef double sd = 1
+    #cdef int n_up = 0
+    #cdef int n_low = 0
+    #cdef double sd_up
+    #cdef double sd_low
+    cdef double p
+    cdef double sum_logp = 0
+    cdef double wp_outlier = w_outlier * p_outlier
+    cdef double alfa 
+    cdef double neg_alpha = (2.718281828459**alpha)/(1+2.718281828459**alpha)
+    cdef double pos_alpha = 2.718281828459**(alpha + dual_alpha)/(1+2.718281828459**(alpha + dual_alpha))
+    cdef np.ndarray feedbacks
```

Comment:
If you know the dtype it'd be faster if you specified it here, e.g. `cdef np.ndarray[double] feedbacks`